frontend/src/components/AssignmentDialog/ContactsInfoCard.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-col cols="12" sm="6">
+    <v-window v-model="contactPage" show-arrows="hover">
+      <v-window-item v-for="contact in assignment.contacts" :key="contact.id">
+        <v-card position="relative" height="380" class="contact-info-wrapper">
+          <div class="info-wrapper">
+            <v-chip class="mr-5" color="blue">
+              {{ contactPage + 1 }} / {{ assignment.contacts.length }}
+            </v-chip>
+
+            <v-btn
+              v-if="userState.role == ROLES.insurance"
+              class="ml-5"
+              color="orange"
+              density="comfortable"
+              icon="mdi-pencil"
+              @click="openEditContactModal(contact)"
+            > 
+              <v-icon color="white" icon="mdi-pencil" />
+              <v-tooltip
+                activator="parent"
+                text="Edit contact"
+                location="bottom"
+              />
+            </v-btn>
+
+            <v-btn
+              v-if="userState.role == ROLES.insurance && !isExcludedContactTypes(contact.type)"

```

Comment:
```suggestion
              v-if="userState.role === ROLES.insurance && !isExcludedContactTypes(contact.type)"

```

Please use tripple equality instead

Reply:
Done